### PR TITLE
fixed 'null is not an object (evaluating 'this._map.containerPointToLayerPoint')'

### PR DESCRIPTION
**Pull request by smeyer-p2er**

### CODE REVIEW

File: leaflet-maplibre-gl.js [hunk 1]
```diff
@@ -155,6 +155,9 @@
         },
 
         _update: function (e) {
+            if (!this._map) {
+                return;
+              }
```
Comment (ianthetechie) [inline leaflet-maplibre-gl.js:160]:
```suggestion
            }
```

**Review by ianthetechie — CHANGES_REQUESTED** [review]
Thanks for your PR @smeyer-p2er! Content-wise this looks good to me. Just one minor nit with formatting.

Two quick things to streamline future contributions:

* Could you enable maintainer edits on your PRs? That way we can make these changes and merge faster!
* You can add "Closes # ..." to your PR description to auto-close issues too!

**Comment by smeyer-p2er** [comment]
Thanks for your review. I'm not sure how to enable "maintainer edits" .

**Review by ianthetechie — APPROVED** [review]

**Comment by ianthetechie** [comment]
@lseelenbinder can you merge + push this? Looks like I don't have write access ;)

**Review by lseelenbinder — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**